PartEzApp/app/src/main/java/com/example/gregjoubert/partezapp/HomeActivity.java

```diff
@@ -10,15 +10,29 @@
 import android.text.TextUtils;
 import android.util.Log;
 import android.view.View;
+import android.widget.Button;
+import android.widget.ExpandableListView;
 import android.widget.TextView;
 import android.widget.Toast;
+
+import com.example.gregjoubert.partezapp.DataWrapper.Result;
+import com.example.gregjoubert.partezapp.DataWrapper.SearchResponse;
```

Comment:
We should probably do away with the gregjoubert packages before we develop further.